terraform/bod_docker_cloud_init.tf

```diff
@@ -38,6 +38,39 @@ data "cloudinit_config" "bod_docker_cloud_init_tasks" {
     filename     = "vdp_disk_setup.sh"
   }
 
+  part {
+    content = templatefile("${path.module}/cloud-init/chown_directory.tpl.sh", {
+      group          = "cyhy"
+      is_mount_point = true
+      owner          = "cyhy"
+      path           = "/var/cyhy/orchestrator/output"
+    })
+    content_type = "text/x-shellscript"
+    filename     = "cyhy_docker_chown_orchestrator_output_directory.sh"
+  }
+
+  part {
+    content = templatefile("${path.module}/cloud-init/chown_directory.tpl.sh", {
+      group          = "cyhy"
+      is_mount_point = true
+      owner          = "cyhy"
+      path           = "/var/cyhy/vdp/output"
+    })
+    content_type = "text/x-shellscript"
+    filename     = "cyhy_docker_chown_vdp_output_directory.sh"
+  }
+
+  part {
```

Comment:
Would you re-arrange these `part`s so they are ordered how we expect them to run on the instance?

Reply:
I can certainly reorder the script `part`s in that order.  I believe the order in which the various `text/cloud-config` bits run is dependent on the contents and has nothing to do with the `part`s.  One such `part` could contain two pieces of content that are run at different stages, but for the most part they will all run _before_ the user scripts.

Shall I just put all the `text/cloud-config` pieces at the top of the file (ordered by their `filename` attribute) and all the `text/x-shellscript` parts below that (ordered according to how they are expected to run on the instance)?

Reply:
Yes you're right about `text/cloud-config` ordering and I'm totally fine with your solution as that's what I was thinking myself.

Reply:
Please see commit ddfa965.